src/main/resources/application.yml

```diff
@@ -16,7 +16,7 @@ spring:
   mail:
     host: smtp.gmail.com
     port: 587
-    username: prisoner.transactions@digital.justice.gov.uk
+    username: prisoner.transactiions@gmail.com
     password: fakePassword
```

Comment:
Double ii ?

Reply:
Yes I'm afraid so. It's just for the POC.